handlers/api.go

```diff
@@ -1019,18 +1019,17 @@ func ApiDashboard(w http.ResponseWriter, r *http.Request) {
 	sendOKResponse(j, r.URL.String(), []interface{}{data})
 }
 
-func getSyncCommitteeFor(validators []uint64, period uint64) ([]interface{}, error) {
+func getSyncCommitteeInfoForValidators(validators []uint64, period uint64) ([]interface{}, error) {
 	rows, err := db.ReaderDb.Query(
 		`SELECT 
 			period, 
-			period*$2 AS start_epoch, 
-			(period+1)*$2-1 AS end_epoch, 
 			ARRAY_AGG(validatorindex ORDER BY committeeindex) AS validators 
 		FROM sync_committees 
-		WHERE period = $1 AND validatorindex = ANY($3)
+		WHERE period = $1 AND validatorindex = ANY($2)
 		GROUP BY period`,
 		period,
 		utils.Config.Chain.ClConfig.EpochsPerSyncCommitteePeriod,
+		utils.Config.Chain.ClConfig.AltairForkEpoch,
 		pq.Array(validators),
```

Comment:
Well, this ain't good. Will be fixed in https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2622